### DISCUSSION TITLE
How to wait until pods are ready with autoPromotionEnabled: false ?

### DISCUSSION THREAD

**Discussion by jr01** [argoproj/argo-rollouts#2098] [Q&A]
I'm brand new to Argo rollouts and trying blue/green with manual pause.

I modified [examples/helm-blue-green](https://github.com/argoproj/argo-rollouts/tree/master/examples/helm-blue-green/templates/rollout.yaml) and set `autoPromotionEnabled: false` within the blueGreen strategy.

Then I deploy the 2nd version: `helm upgrade example .  --set image.tag=0.2`

And now I want to wait until the 2nd version is running and then 'test' it. But this: 
```
> kubectl argo rollouts status example-helm-guestbook
Paused - BlueGreenPause
```
just keeps waiting.

How can I wait until the pods are ready and the `BlueGreenPause` state is reached?

**Comment by kostis-codefresh** [comment]
Hello

"Paused" refers to the whole Rollout and not the pods of the second color. The pods  should be up after some time.

Also if you want to "test" the pods you should not do it manually but instead use an Analysis like https://argoproj.github.io/argo-rollouts/analysis/job/
